pom.xml

```diff
@@ -36,8 +36,8 @@
   </description>
 
   <properties>
-    <jackrabbit.version>2.18.2</jackrabbit.version>
-    <oak.version>1.22.1</oak.version>
+    <jackrabbit.version>2.20.6</jackrabbit.version>
```

Comment:
AEM 6.5 (15) is still on 2.20.4, but if it is semantically versioned that shouldn't matter....

Reply:
I don't think that comment was actionable.